index.html

```diff
@@ -118,6 +123,17 @@
             "!WEBIDL#securityerror"><code>SecurityError</code></a>, and abort
             these steps.
             </li>
+            <li>Let <var>url</var> be the result of running the <a data-cite=
```

Comment:
FYI: I've filed a Chrome bug [crbug.com/734486](https://crbug.com/734486) for updating Chrome's behaviour.